client/src/components/Panel/Panel.stories.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Panel } from "./Panel";
+
+export default {
+  title: "Panel",
+  component: Panel,
+};
+
+const Template = (args) => <Panel {...args} />;
+
+const children = <div>Children</div>;
+
+export const Basic = Template.bind({});
+Basic.args = {
+  allowOverflow: true,
+  title: "title",
+  otherHeaderContent: "",
+  children,
+  sources: "",
+  id: 1,
+  glossary_keys: {},
+  footnotes: [],
+  isOpen: true,
+  style: {},
+};
```

Comment:
Another tricky one, lots of props with very specific infobase meaning. Also note sure `glossary_keys` can be used (without some trickery), as the glossary data won't be populated within the storybooks, hmm.

I think I'd advice dropping this story for now. It deserves its own PR, shouldn't hold this one up..